.env.example

```diff
@@ -1 +1,2 @@
+CHROMATIC_PROJECT_TOKEN=14f589a59855
```

Comment:
Although is not ideal, we agreed on having this token shared among contributors to allow pull request checks from forks.

More info: https://www.chromatic.com/docs/github-actions#run-chromatic-on-external-forks-of-open-source-projects